R/attachment.R

```diff
@@ -12,10 +13,11 @@
 #' attachment(msg, "cat.png")
 #' attachment(msg, "visualisations.png", "visuals")
 #' }
-attachment <- function(msg, path, cid = NA, disposition = NA){
+attachment <- function(msg, path, cid = NA, disposition = NA, mime_type){
   if (length(path) != 1) stop("Must be precisely one attachment.", call. = F)
 
   type <- guess_type(path, empty = NULL)
```

Comment:
```suggestion
  type <- guess_type(path, empty = "application/octet-stream")
```

If you agree, we can add this change to this PR.